x-pack/plugins/infra/public/components/logging/log_flyout.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import React from 'react';
+import styled from 'styled-components';
+import { InfraLogItem, InfraLogItemField } from '../../graphql/types';
+import { InfraLoadingPanel } from '../loading';
+interface Props {
+  flyoutItem: InfraLogItem;
+  hideFlyout: () => void;
+  setFilter: (filter: string) => void;
+  intl: InjectedIntl;
+  loading: boolean;
+}
+
+export const LogFlyout = injectI18n(
+  ({ flyoutItem, loading, hideFlyout, setFilter, intl }: Props) => {
+    const handleFilter = (field: InfraLogItemField) => () => {
+      const filter = `${field.field}:"${field.value}"`;
+      setFilter(filter);
+    };
+
+    const columns = [
+      {
+        field: 'field',
+        name: intl.formatMessage({
+          defaultMessage: 'Field',
+          id: 'xpack.infra.logFlyout.fieldColumnLabel',
+        }),
+        sortable: true,
+      },
+      {
+        field: 'value',
+        name: intl.formatMessage({
+          defaultMessage: 'Value',
+          id: 'xpack.infra.logFlyout.valueColumnLabel',
+        }),
+        sortable: true,
+        render: (name: string, item: InfraLogItemField) => (
+          <span>
+            <EuiToolTip
+              content={intl.formatMessage({
+                id: 'xpack.infra.logFlyout.setFilter',
```

Comment:
If I read the i18n guidelines correctly, every id should end with a type. Might be

```suggestion
                id: 'xpack.infra.logFlyout.setFilterTooltip',
```

in this case?